pipeline/talend_test/ansible/roles/talend_test/library/allowed_dir.py

```diff
@@ -0,0 +1,47 @@
+
+import os
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def prep_dir(dir_or_file):
+    if '.' in os.path.basename(dir_or_file):
+        return os.path.dirname(dir_or_file)
+    else:
+        return os.path.dirname(dir_or_file)
```

Comment:
Looks to return exactly the same thing no matter what

Reply:
thanks. I've fixed now